docs/BreakingChanges/113 Intermittently unable to scroll to bottom item in ItemsControls.md

```diff
@@ -0,0 +1,35 @@
+## 113: Intermittently unable to scroll to bottom item in ItemsControls (like ListBox and DataGrid) when using custom DataTemplates
```

Comment:
{
  "Id": "113",
  "Title": "Intermittently unable to scroll to bottom item in ItemsControls (like ListBox and DataGrid) when using custom DataTemplates",
  "Details": "In some instances, a bug in the .NET Framework 4.5 is causing ItemsControls (like ListBox, ComboBox, DataGrid, etc.) to not scroll to their bottom item when using custom DataTemplates. If the scrolling is attempted a second time (after scrolling back up), it will work then.",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.5.2 and may be addressed by upgrading to that version (or a later version) of the .NET Framework. Alternatively, users can still drag scroll bars to the final items in these collections, but may need to try twice to do so successfully.",
  "Link": "",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.2",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/759702",
  "Notes": "This is difficult to detect with ApiPort because we can't know which of all ItemsControl instances have custom data templates that would make them susceptible to this. We should learn more about the issue to understand whether or not an analyzer could do a better job.\nSource analyzer status: Pri 3, Investigate",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis